src/process_utility.c

```diff
@@ -1443,6 +1460,28 @@ process_altertable_start_table(Node *parsetree)
 				if (ht != NULL)
 					process_altertable_drop_not_null(ht, cmd);
 				break;
+			case AT_AddColumn:
+			case AT_AddColumnRecurse:
+				{
+					ColumnDef  *col;
+					ListCell   *_lc;
```

Comment:
Think this should just be `*lc`, no underscore

Reply:
There already is an `lc` in a higher scope that is visible here.
I think it will work if I redeclare it but is likely to create weird bugs later on when this code changes.

Reply:
Ah gotcha, good idea.

Reply:
should be constraint_lc